Resources/Prototypes/Entities/Objects/Specific/Magical/SpellBooks.yml

```diff
@@ -0,0 +1,18 @@
+- type: entity
+  name: Urban Legends
+  parent: BaseItem
+  id: Debug Spellbook
+  description: 'penned by "Beyond Occult Rituals Typography"'
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/books.rsi
+    state: bookold
+  - type: SpellBook
+    usemessage: You're a wiznred, Garry!
```

Comment:
1 example of the spelling mistakes.

Reply:
oh that's actually just a "You're are a wizard, Harry!" but jumbled

Reply:
Maybe change it to wiznerd?

Reply:
Oh sorry, my bad